cmd/tealdbg/cdtdbg.go

```diff
@@ -84,6 +85,7 @@ func (a *CdtFrontend) SessionStarted(sid string, debugger Control, ch chan Notif
 	s.states = debugger.GetStates(nil)
 
 	a.sessions[sid] = *s
+	a.latestSid = sid
```

Comment:
maybe I'm missing something here: it looks as if a call to `SessionEnded` could leave you with a `a.latestSid` that doesn't have a matching session object, causing `URL()` to return an empty string ?

Reply:
Your are right, although this implementation works for a current use scenario, it breaks in others. I'll fix it.

Reply:
changed to ordered array of session ids, return the last.